### ISSUE TITLE
plugin doesn't work?

### ISSUE THREAD

**Issue by omercnet** [pulumi/eslint-plugin-pulumi#11]
### What happened?

tried a simple test using the example and it didn't work

### Steps to reproduce

https://github.com/omercnet/pulumi-eslint-miss

### Expected Behavior

should warn

### Actual Behavior

warnings from other plugins but not pulumi

```
  10:7  warning  'someInterpolatedString' is assigned a value but never used  @typescript-eslint/no-unused-vars
  12:7  warning  'input' is assigned a value but never used                   @typescript-eslint/no-unused-vars
```

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by RobbieMcKinstry** [comment]
Hmm, the gist you provided looks reasonable to me, and I'm not aware of any changes that would the linter uninstallable. I'll need to attempt a reproduction using your gist.

**Comment by blakegong** [comment]
Hey @omercnet I bumped into a similar issue, and your mini repo was a perfect place to test this ESLint plugin out. Here's the fix:

<img width="1176" alt="image" src="https://user-images.githubusercontent.com/3624869/220604010-7429b0a6-1dc3-4450-9136-e09c09cd9fc0.png">

2 things need fixing:

1. `@pulumi/no-output-in-template-literal` is just not a warning/error by default 🫠
2. After it is enabled, you'd get an error: `Error: Error while loading rule '@pulumi/no-output-in-template-literal': You have used a rule which requires parserServices to be generated. You must therefore provide a value for the "parserOptions.project" property for @typescript-eslint/parser.`. I used [this comment](https://github.com/prettier/prettier-eslint/issues/201#issuecomment-901299351) for a quick fix, which led to the line on `"project": ["tsconfig.json"] ,`

Result:

```
➜  pulumi-eslint-miss git:(main) ✗ npm run lint

> lint
> eslint .


/Users/blake/Code/opengovsg/pulumi-eslint-miss/index.ts
  10:7   warning  'someInterpolatedString' is assigned a value but never used  @typescript-eslint/no-unused-vars
  10:32  error    pulumi.Output<T> not permitted in template literals          @pulumi/no-output-in-template-literal
  12:7   warning  'input' is assigned a value but never used                   @typescript-eslint/no-unused-vars
  12:45  error    pulumi.Output<T> not permitted in template literals          @pulumi/no-output-in-template-literal

✖ 4 problems (2 errors, 2 warnings)
  2 errors and 0 warnings potentially fixable with the `--fix` option.
```

I believe the real fix for point 2 is in this repo, but I didn't have time to look into the details. I'm also wishing this repo will just provide a working sample `.eslintrc` file so that we won't have to jump through these hoops in the future 😅.

**Comment by pimvanderheijden** [comment]
Yeah and `.eslintrc.json` please 😃
